sealedenumprocessor/build.gradle.kts

```diff
@@ -14,6 +14,7 @@ dependencies {
 
     testImplementation(Dependencies.junit)
     testImplementation(Dependencies.mockk)
+    testImplementation(kotlin("reflect"))
```

Comment:
Huh...what's going on here? Why the String?

Reply:
It's a separate shorthand for `kotlin` related dependencies, and I believe it pulls in the version automatically from the `kotlin` plugin.